src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -899,20 +902,19 @@ public void testPrintSuppressionOption() throws Exception {
 
     @Test
     public void testPrintSuppressionAndTabWidthOption() throws Exception {
-        final String expected = "/CLASS_DEF[./IDENT[@text='InputMainSuppressionsStringPrinter']]"
+        final String expected = addEndOfLine(
+            "/CLASS_DEF[./IDENT[@text='InputMainSuppressionsStringPrinter']]"
                 + "/OBJBLOCK"
+                + "/METHOD_DEF[./IDENT[@text='getName']]/SLIST/VARIABLE_DEF[./IDENT[@text='var']]",
+                "/CLASS_DEF[./IDENT[@text='InputMainSuppressionsStringPrinter']]/OBJBLOCK"
                 + "/METHOD_DEF[./IDENT[@text='getName']]/SLIST/VARIABLE_DEF[./IDENT[@text='var']]"
-                + EOL
-                + "/CLASS_DEF[./IDENT[@text='InputMainSuppressionsStringPrinter']]/OBJBLOCK"
+                + "/MODIFIERS",
```

Comment:
for some cases it is impossible, it will break line length rule. It can be reverted to old way or some kind of exception is added to linelength.
What do you think?

Reply:
changed

Reply:
changed